sw/device/sca/lib/sca.c

```diff
@@ -253,6 +267,14 @@ void sca_set_trigger_low() {
 }
 
 void sca_call_and_sleep(sca_callee callee, uint32_t sleep_cycles) {
+  // Disable the IO_DIV4_PERI clock to reduce noise during the actual capture.
+  // This also disables the UART(s) and GPIO modules required for
+  // communication with the scope. Therefore, it has to be re-enabled after
+  // the capture.
+  dif_clkmgr_t clkmgr;
+  OT_DISCARD(dif_clkmgr_init(
+      mmio_region_from_addr(TOP_EARLGREY_CLKMGR_AON_BASE_ADDR), &clkmgr));
```

Comment:
nit: not sure if this makes a difference at all but we may want to make `clkmgr` static instead of initializing both here and in `sca_disable_peripherals()`.

Reply:
This sounds like a good suggestion but I think I am switching too much between programming languages at the moment... would it work to just declare a `static dif_clkmgr_t clkmgr;` in this file or would we need to do that in the `aes/sha3_serial.c` and then add a `clkmgr` argument to both these functions inside `sca.c`?

Reply:
No worries, I meant the former, i.e. in this file (similar to the declarations in lines 57-60).